static/src/javascripts/projects/common/modules/identity/email-preferences.js

```diff
@@ -30,6 +29,7 @@ define([
                     if (response && response.subscriptions && response.subscriptions.length) {
                         isSubscribed = true;
                     }
+                    $(buttonEl).removeClass('js-button-updating');
```

Comment:
On membership we had a convention that state classes began with `.is-`, e.g. `.is-updating`. The `.js-` classes were to provide hooks for JS behaviour, so in this case `if ($(buttonEl).hasClass('save__button'))` is where you would use a `.js-` class to distinguish from classes used for styling. The idea being that you should be able to change a non `.js-` class without worrying that it would break some behaviour.